api/WEBGL_compressed_texture_etc.json

```diff
@@ -29,10 +29,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": null
+            "version_added": false
```

Comment:
This is an unusual situation, I guess, but http://mdn-bcd-collector.appspot.com/tests/api/WEBGL_compressed_texture_etc does show false on Safari 14 on a physical MBP.